# Study 933 — Same Issuer, Two Ladders 🪜

**For one and the same balance sheet, does the junior rung pay you for standing behind the senior one?**

Eight US issuers list **two** $25-par instruments side by side: a **preferred** (the junior
rung — discretionary coupon, usually no maturity, behind every debt claim) and an
exchange-traded **baby bond**. Textbook risk pricing says the preferred must pay more. This
study holds the **obligor fixed** and measures how much.

*(The rungs are not uniform: only **2 of 8** pairs are the textbook
"perpetual preferred vs *senior* note" — in four the note is itself junior subordinated or
perpetual, and in two the "preferred" is a dated term preferred. A one-rung step cannot pay a
two-rung premium; `data.RUNG_STRUCTURE` names each one.)*

The pairs: CMS Energy, Duke Energy, Brookfield Renewable, Brookfield Infrastructure,
B. Riley Financial, Oxford Lane Capital, Eagle Point Credit, Babcock & Wilcox — daily
**total-return** closes, 2022-02-02 → 2026-06-30 (1,105 days), two equal-weight
ladders rebalanced monthly with **one execution lag**, 25 bps one-way, both
**excess-of-cash** (BIL).

*Numbers below are the frozen headline (`docs/results.md`, Fingerprint `9ae51d88019e`); the live
cells run only the fast synthetic control. As-of 2026-06-30.*


## 1. Two rungs of one ladder

Picture a company that owes money in two publicly listed ways. The **baby bond** is debt: a fixed maturity, a contractual coupon, and a legal queue position ahead of shareholders. The **preferred** is the rung below: no maturity date, a coupon the board can switch off, and a place in the queue behind every lender. Same company, same trucks and power plants and loan book — different seat on the lifeboat.

So the preferred should pay you extra. The question is how much, and whether four and a half years of tape can actually see it.

In [1]:
R = dict(pref_sharpe=0.368, bond_sharpe=0.202, pref_cagr=5.35, bond_cagr=1.82,
         pref_vol=18.96, bond_vol=13.31, pref_dd=-30.82, bond_dd=-20.4,
         pairwise=3.1, t_pairwise=0.55, vol_ratio=1.42,
         ex2_pw=-0.6, ex2_ratio_m=0.79)
print('Preferred ladder (junior): %+.2f%% a year over cash, %.1f%% swings, worst fall %.1f%%'
      % (R['pref_cagr'], R['pref_vol'], R['pref_dd']))
print('Baby-bond ladder (senior): %+.2f%% a year over cash, %.1f%% swings, worst fall %.1f%%'
      % (R['bond_cagr'], R['bond_vol'], R['bond_dd']))
print()
print('extra paid for standing further back: %+.2f%% a year  (t = %+.2f)'
      % (R['pairwise'], R['t_pairwise']))
print('extra risk taken for it            : %.2fx the volatility'  % R['vol_ratio'])
print()
print('...but drop the two distressed issuers and BOTH numbers invert:')
print('   paid %+.2f%% a year, and %.2fx the volatility (i.e. the CALMER leg)'
      % (R['ex2_pw'], R['ex2_ratio_m']))

Preferred ladder (junior): +5.35% a year over cash, 19.0% swings, worst fall -30.8%
Baby-bond ladder (senior): +1.82% a year over cash, 13.3% swings, worst fall -20.4%

extra paid for standing further back: +3.10% a year  (t = +0.55)
extra risk taken for it            : 1.42x the volatility

...but drop the two distressed issuers and BOTH numbers invert:
   paid -0.60% a year, and 0.79x the volatility (i.e. the CALMER leg)


## 2. The payment is a rumour

At first glance the numbers look like the textbook: the junior ladder swung **1.42×** as hard, fell **-30.8%** at worst against **-20.4%**, and was the more volatile leg in **6 of 8** pairs — more risk, and **+3.10%/yr** more return to go with it.

Both halves of that sentence fall apart on inspection. Start with the return: **+3.10%/yr** sounds like a premium until you see the error bars — the plausible range runs from **-6.8%** to **+13.6%**. That is not a measurement, it is a shrug. (Sections 3 and 4 take the risk half apart.)

> 🔬 *For the quants:* HAC (Newey-West) *t* = **+0.55** on the pairwise daily difference; the Sharpe advantage **+0.166** carries a joint block-bootstrap CI of **[-0.47, +0.80]**, with **32%** of resamples negative. And note what the pairwise spread *is*: an arithmetic mean of daily differences, re-equal-weighted every day and charged no cost — a statistic, not a portfolio.

## 3. Two companies are carrying the whole average

**B. Riley Financial** went through a genuine distress episode and back. Its preferred crashed and rebounded so violently that, averaged day by day, it contributes a **+19.1%/yr** spread — enough on its own to make the panel look like it pays a premium. Take it out and the number falls to **+0.81%/yr** (*t* = +0.17).

Take out **Babcock & Wilcox** as well — the other issuer that spent this window in trouble — and the premium doesn't just shrink, it **changes sign**: **-0.60%/yr** across the remaining 6 pairs, with a Sharpe advantage of **-0.116**. The typical pair pays **+0.70%/yr** — about a fifth of a percent a quarter, for standing behind every lender.

> 🔬 *For the quants:* and even B. Riley's headline number is a **Jensen trap** — its *arithmetic* daily spread is +19.1%/yr while its *compounded* outcome is the opposite (-7.71%/yr CAGR on the preferred vs +2.58% on the bond). The investor who actually held it lost.

## 4. And the 'at least it's riskier' consolation prize isn't real either

If the junior rung isn't *paid* more, you would at least expect the textbook's other half to hold — that it is reliably the more dangerous seat. It isn't. Two things were hiding inside that **6/8** count.

**These tickers barely trade.** A $25-par preferred can go a whole day without a single print, and then bounce between bid and offer the next. CMS Energy's preferred shows **no price change at all on 15.5%** of days and an up-down-up ping-pong pattern (lag-1 autocorrelation **-0.28**). Measured day by day it looks like a **24.0%**-volatility instrument; measured month by month, where the ping-pong cancels out, it is **11.4%** — *below* its own baby bond's 13.5%. Widen the measuring window and the count falls **6 → 4 → 2** (daily → weekly → monthly).

**And the two survivors are the same two distressed companies.** Drop B. Riley and Babcock & Wilcox and the junior ladder becomes the *calmer* one: volatility ratio **0.79** monthly, junior riskier in **0 of 6** pairs, worst falls of **-20.1%** vs **-19.3%** — indistinguishable.

So the junior rung is the riskier one *when the company is in trouble*. That is a statement about trouble, not about seniority — and it is not something the market was paying you in advance to bear.

## 5. And seniority didn't save you when it mattered

Being senior is supposed to buy protection. On this tape it mostly didn't:

| when it got ugly | preferred | baby bond | who won |
|---|--:|--:|---|
| 2022 rate shock | −18.7% | −18.7% | a dead heat |
| March-2023 bank stress | +0.6% | −2.3% | **the junior rung** |
| 2024 issuer stress | +13.9% | +8.6% | **the junior rung** |
| 2025 tariff shock | −10.6% | −9.5% | the senior rung, barely |
| COVID crash (2 long-history pairs) | **-10.2%** | **-19.8%** | **the junior rung, by 9.5 pp** |

The reason is unglamorous: between two rungs of one issuer, what usually differs most is **duration**, not seniority. A perpetual preferred and a 40-year note are both long; a 2028 note is not. In 2022 and in COVID the market was pricing interest rates, and the 'safer' rung was simply the more rate-sensitive one.

## 6. Two more reasons to distrust the +3.1%

**The screen throws away the counter-examples.** We could only include pairs where *both* rungs are still listed in June 2026 — which quietly deletes every redeemed note and called preferred. The one dead pair we can still measure (Sachem Capital, whose bond matured in December 2024) ran the other way entirely: the preferred returned **-17.4%** while the bond returned **+21.6%** — a spread of **-10.63%/yr**.

**The broad market disagrees with our eight names.** Over exactly this window the whole preferred sector (PFF) **lost 4.75%/yr** to our baby-bond basket. Our preferreds won; preferreds in general did not.

## 7. Live check — the yardstick works (offline synthetic)

Before believing a null result, check the instrument can detect anything at all. We build a fake world of eight issuers where the junior rung genuinely *is* paid an extra 6% a year, and a matched world where it is paid exactly nothing.

In [2]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from two_ladders import data, strategy as st
paid   = st.synthetic_detect(*data.synthetic_panel(signal_strength=1.0, seed=933)[:3])
unpaid = st.synthetic_detect(*data.synthetic_panel(signal_strength=0.0, seed=933)[:3])
print('world where the junior rung IS paid +6.0%%/yr : measured %+.2f%%/yr  (t = %+.2f)'
      % (paid['pairwise_spread_ann']*100, paid['t_pairwise']))
print('world where it is paid nothing               : measured %+.2f%%/yr  (t = %+.2f)'
      % (unpaid['pairwise_spread_ann']*100, unpaid['t_pairwise']))

world where the junior rung IS paid +6.0%/yr : measured +4.27%/yr  (t = +3.88)
world where it is paid nothing               : measured -1.73%/yr  (t = -1.57)


On this single draw the yardstick picks the real premium out clearly (a *t* well past 2) and fails to reach significance in the world where nothing is paid — and across 20 draws (`docs/results.md`) it fires **20/20** times on the planted world and **0/20** on the null. So the silence on the real tape is the tape's, not the measuring stick's.

## Verdict

- **Signal — None.** The seniority premium is not measurable here: **+3.10%/yr** with *t* = **+0.55** and a CI of **[-6.8%, +13.6%]**; the sign flips between the two halves of the sample; it collapses to **+0.81%** without one issuer and to **-0.60%** without two; and over the longer seven-year two-pair tape it is **-0.43%/yr**. Nor is there a consolation risk premium to point at: the junior rung's apparent extra risk is a **thin-tape artefact** plus **two distressed names**, and outside them the junior rung is the calmer leg in **0 of 6** pairs.
- **Tradability — Mirage.** Long the preferreds, short the bonds: **+4.29%/yr** gross at Sharpe 0.28 on a *t* of 0.65, down to Sharpe 0.09 once you assume a realistic borrow cost, with a -26% drawdown along the way. If you want the issuer's credit, the senior rung gave you two-thirds of the return for two-thirds of the risk — and slept better.